src/components/m-accordion-item/_template.js

```diff
@@ -0,0 +1,34 @@
+import bel from 'bel';
+import classnames from 'classnames';
+import { BaseComponent } from '../_abstract/component-types';
+
+export default ({ header, headersecondary, headercolor, classes, multiple, open = false, icon }, childrenFragment) => {
+  const headerPrimaryClasses = classnames('m-accordion-item__header__primary', {
+    [`m-accordion-item__header__primary--${headercolor}`]: headercolor,
+    'm-accordion-item__header__primary--with-secondary': headersecondary,
+  });
+  const headerClasses = classnames('m-accordion-item__header', {
+    'm-accordion-item__header__primary--with-icon': icon,
+  });
+
+  const iconToRender = bel`<axa-icon class="m-accordion-item__header__icon-wrapper" id="${icon}" classes="m-accordion-item__header__icon"></axa-icon>`;
+  const chevron = bel`<axa-icon class="m-accordion-item__header__icon-wrapper" id="chevron-down" classes="m-accordion-item__header__chevron"></axa-icon>`;
+
+  return bel` 
+      <div class="${classes}">
```

Comment:
could be a section or better an article:

Examples of sections would be chapters, the various tabbed pages in a tabbed dialog box, or the numbered sections of a thesis. A Web site's home page could be split into sections for an introduction, news items, and contact information.

Authors are encouraged to use the article element instead of the section element when it would make sense to syndicate the contents of the element.

Reply:
ref: https://www.w3.org/TR/2011/WD-html5-20110525/sections.html#the-section-element